tools/backwardCompatibleKeys.js

```diff
@@ -0,0 +1,10 @@
+const backwardCompatibleKeys = {
```

Comment:
Can we rename this file to backward-compatible-keys.js to match our file naming conventions?

Reply:
Apologies. Done...